xapi-database.opam

```diff
@@ -9,14 +9,22 @@ build: [[ "jbuilder" "build" "-p" name ]]
 
 depends: [
   "jbuilder" {build & >= "1.0+beta11"}
-  "rpc"
+
+  "gzip"
+  "http-svr"
+  "ocaml-migrate-parsetree"
```

Comment:
Omp appears in multiple packages but it should not, maybe something is wrong in some dune/jbuilder file

Reply:
We should keep an eye out when updating the other packages, eventually we should find the one responsible for it.